src/Tickets/Commerce/Gateways/PayPal/Buttons.php

```diff
@@ -78,60 +66,11 @@ public function get_checkout_script() {
 			$template_vars['client_token'] = $client_token;
 			$template_vars['client_token_expires_in'] = $client_token_expires_in - 60;
 		}
-
-		$html = $this->get_template()->template( 'checkout-script', $template_vars, false );
-
-		tribe_asset_enqueue( 'tec-tickets-commerce-gateway-paypal-checkout' );
-
-		return $html;
-	}
-
-
-	/**
-	 * Include the payment buttons from PayPal into the Checkout page.
-	 *
-	 * @since 5.2.0
-	 *
-	 * @param string           $file     Which file we are loading.
-	 * @param string           $name     Name of file file
-	 * @param \Tribe__Template $template Which Template object is being used.
-	 *
-	 */
-	public function include_payment_buttons( $file, $name, $template ) {
-		$must_login = ! is_user_logged_in() && tribe( Module::class )->login_required();
-
-		$template->template( 'gateway/paypal/buttons', [ 'must_login' => $must_login ] );
-	}
-
-
-	/**
-	 * Include the advanced payment fields from PayPal into the Checkout page.
-	 *
-	 * @since 5.2.0
-	 *
-	 * @param string           $file     Which file we are loading.
-	 * @param string           $name     Name of file file
-	 * @param \Tribe__Template $template Which Template object is being used.
-	 *
-	 */
-	public function include_advanced_payments( $file, $name, $template ) {
-		$items = tribe( Cart::class )->get_items_in_cart( true );
-
-		// Bail if there are no tickets in cart.
-		if ( empty( $items ) ) {
-			return;
-		}
-
-		$must_login = ! is_user_logged_in() && tribe( Module::class )->login_required();
-		$merchant   = tribe( Merchant::class );
-
-		$template->template(
-			'gateway/paypal/advanced-payments',
-			[
-				'supports_custom_payments' => $merchant->get_supports_custom_payments(),
-				'active_custom_payments'   => $merchant->get_active_custom_payments(),
-				'must_login'               => $must_login,
-			]
-		);
```

Comment:
Why was this removed? Why are we touching PayPal codebase?

Reply:
I consolidated the several different template methods into one `get_checkout_template_vars()` method which returns the variables needed for the templates. Rather than using hooks (in the `Hooks.php` file), I decided to add a main gateways template (`src/views/v2/commerce/gateway/gateways.php`) that calls methods from each gateway to pull the template info to show it. I felt it gave us more flexibility with the templates and reduced the amount of calls it required.